src/main/java/org/ays/auth/filter/AysBearerTokenAuthenticationFilter.java

```diff
@@ -1,41 +1,40 @@
 package org.ays.auth.filter;
 
-import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
-import com.google.common.cache.LoadingCache;
-import io.github.bucket4j.Bandwidth;
-import io.github.bucket4j.Bucket;
 import jakarta.servlet.FilterChain;
 import jakarta.servlet.ServletException;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
 import jakarta.validation.constraints.NotNull;
 import lombok.RequiredArgsConstructor;
-import lombok.extern.slf4j.Slf4j;
-import org.ays.auth.model.AysToken;
 import org.ays.auth.service.AysInvalidTokenService;
 import org.ays.auth.service.AysTokenService;
-import org.ays.common.util.HttpServletRequestUtil;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.http.HttpHeaders;
+import org.ays.common.model.request.AysHttpHeader;
+import org.ays.common.model.request.AysHttpServletRequest;
+import org.ays.common.model.response.AysHttpServletResponse;
+import org.springframework.core.annotation.Order;
 import org.springframework.lang.NonNull;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Component;
 import org.springframework.web.filter.OncePerRequestFilter;
 
 import java.io.IOException;
-import java.time.Duration;
-import java.util.List;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
 
 /**
- * AysBearerTokenAuthenticationFilter is a filter that intercepts HTTP requests and processes the Bearer tokens included in the Authorization headers.
- * If the token is valid, the user is authenticated and added to the SecurityContext for the duration of the request.
- * If the token is invalid, a 401 Unauthorized response is returned.
- * <p>The filter uses an instance of AysTokenService to verify and validate the token and retrieve the user authentication.
+ * Filter responsible for handling Bearer token-based authentication in incoming HTTP requests.
+ * <p>
+ * This filter intercepts requests and checks for the presence of a Bearer token in the `Authorization` header.
+ * If a valid token is found:
+ * <ul>
+ *     <li>The token is verified and validated using {@link AysTokenService}.</li>
+ *     <li>The token's payload is retrieved, and its validity is checked using {@link AysInvalidTokenService}.</li>
+ *     <li>An {@link org.springframework.security.core.Authentication} object is created and added to the {@link SecurityContextHolder} for the current request.</li>
+ * </ul>
+ * If the token is invalid, the filter does not authenticate the request, and a 401 Unauthorized response may be returned.
+ * <p>
+ * This filter ensures that only authenticated users can access secured resources in the application.
+ * It runs with an order of {@code 3}, which allows it to be executed after other filters with lower order values.
  */
-@Slf4j
+@Order(3)
```

Comment:
@Slf4j yerine @Order(3) kullanılmasının sebebi nedir ?

Reply:
`@Slf4j` anotasyonunu kullanmıyorduk o yüzden kaldırdım. `@Order(3)` de kendi yazdığımız filter'ları kendi içerisinde çalışma sıralarını belirleyebilmek için kullandım. Bu sebeple aslında yerine kullanmadım ikisinin ayrı işlevleri var :